packages/platform/generators.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "name": "nx-plugin",
+  "version": "0.0.1",
+  "generators": {
+    "app": {
+      "factory": "./src/lib/nx-plugin/generators/app/generator",
+      "schema": "./src/lib/nx-plugin/generators/app/schema.json",
+      "description": "Generates an AnalogJs application"
+    }
+  }
```

Comment:
suggestion: let's also add a `schematics` property for AngularCLI users as well

Reply:
I will make the changes but will leave a comment for you to double check this once I push them! Haven't worked with the Angular CLI and schematics besides using them 🙈

Reply:
@goetzrobin you can use a `convertNxGenerator` from `@nrwl/devkit'` I think 🤔

Reply:
You can use `convertNxGenerator()` to do this. https://github.com/angular-threejs/angular-three/blob/main/libs/plugin/src/generators/init/compat.ts#L4
then point the `schematics` to the compat file https://github.com/angular-threejs/angular-three/blob/main/libs/plugin/generators.json#L14